catkin_virtualenv/src/dh_virtualenv/deployment.py

```diff
@@ -170,9 +183,11 @@ def install_dependencies(self):
         # along lines of setuptools), but that does not get installed
         # by default virtualenv.
         if self.upgrade_pip:
+            # (pbovbel) allow pinning the pip version
+            pip_package = 'pip==' + self.pip_version if self.pip_version else 'pip'
             # First, bootstrap pip with a reduced option set (well-supported options)
-            print(self.pip_preinstall_prefix + self.pip_upgrade_args + ['-U', 'pip'])
-            subprocess.check_call(self.pip_preinstall_prefix + self.pip_upgrade_args + ['-U', 'pip'])
+            print(self.pip_preinstall_prefix + self.pip_upgrade_args + ['-U', pip_package])
```

Comment:
Easier to keep these strings in sync if you assigned it to a variable and used it for both, but I recognize that means diverging from upstream.

Reply:
That's the concern